docs/source/quick start.rst

```diff
@@ -3,31 +3,31 @@
+* **ASS format**. As PyonFX is an advanced tool for typesetting and karaoke, it is meant to be used by experienced typesetters who are familiar with the tags Libass supports, as well as how they function. Check the footnote [#f1]_ for a complete list of tags.
+* **Python3 scripting language**. A programming language like Python allows you to define functions for your effects, how many times to perform, what to perform it on based on conditions, etc. In essense, it gives you much more freedom, as you aren't tied to buttons or sliders but instead get to define everything exactly how you want it. **You only need to know the basics for this module**. Knowledge on how to use variable, functions, conditions, loops, comparisons, string formatting, lists, and dictionaries is more than enough. You can find a link to some good tutorials in the footnotes[#f2]_.
 
-To start generating, you will only have to write a script in Python3, which will describes the process of your KFX or advanced typesetting creation.
+To generate the effects, you'll have to write a Python3 script. Within it you will fully define the process of your KFX or advanced typesetting creation.
 
-If you have trouble with the installation of Python, you can check some online guide, like https://realpython.com/installing-python/.
+If you run into problems while installing Python3, there are resources online to help you out, like https://realpython.com/installing-python/ for example.
 
 Windows
 +++++++
 
-So, if you've not installed it before, you will have to **install Python3**.
-You can **download** it from the `official website <https://www.python.org/downloads/>`_.
-Just be sure to check the box that says "Add Python 3.x to PATH". This is really important to avoid some extra steps that would make Python callable in every directory from the command prompt.
+If you haven't installed it yet, make sure to **install** Python3.
+You can **download** it from the `official website: <https://www.python.org/downloads/>`_.
+Make sure you check the box that says "Add Python 3.x to PATH". This is very important to avoid some extra steps that would make Python callable in every directory from the command prompt.
 
 
-Run this command below, which will use pip to install and eventually update the library:
+Run the following command below. It will use pip3 to install and update the library:
 
 .. code-block:: sh
    :emphasize-lines: 1
 
-   pip install --upgrade https://github.com/CoffeeStraw/PyonFX/zipball/master
+   pip3 install --upgrade https://github.com/CoffeeStraw/PyonFX/zipball/master
```

Comment:
It is generally safer to have people use `pip3` and `Python3` whenever possible, just in case some newer users don't recognise the difference and accidentally have Python2 installed somehow.

Reply:
Makes sense 👍